src/dialog/improveAbilityDialog.ts

```diff
@@ -8,7 +8,7 @@ import { assertItemType, getGame } from "../system/util";
 export async function showImproveAbilityDialog(
   actor: RqgActor,
   itemId: string,
-  item: Item,
+  item: RqgItem,
```

Comment:
I merged master into issue97 earlier today and this wasn't building in issue97, though it built fine in master as "Item".  I believe the problem was at line 72 "await submitImproveAbilityDialog(...." where if item is Item it wasn't working but if item is RqgItem it works fine. I think it actually makes sense that it's an RqgItem and it builds and works that way.  This is not related to the default icons issue.  Not sure if this is pointing out some other issue or not.